extensions/dicom-pdf/package.json

```diff
@@ -37,7 +37,8 @@
   "dependencies": {
     "@babel/runtime": "^7.5.5",
     "classnames": "^2.2.6",
-    "lodash.isequal": "^4.5.0"
+    "lodash.isequal": "^4.5.0",
+    "pdfjs-dist": "^2.2.228"
```

Comment:
👀

Reply:
Had to sanity check for myself. Much more usage than `pdfjs` package.
`13.7 MB` -- Holy cow

Reply:
This one comes in at `400kb` https://github.com/wojtekmaj/react-pdf
?

Reply:
> This one comes in at `400kb` https://github.com/wojtekmaj/react-pdf
> ?

@dannyrb I'm not so sure, it uses pdf-dist under the hood:

<img width="358" alt="Screen Shot 2019-12-13 at 09 53 39" src="https://user-images.githubusercontent.com/13886933/70801764-87da1780-1d8e-11ea-80aa-ae87cc381f5c.png">

Reply:
Ah. I was going off the value I found on NPM's website: https://www.npmjs.com/package/react-pdf

It would be odd if it didn't factor in dependencies. I'm curious if it does anything fancy to reduce the bundle's size, or handles code splitting/workers on our behalf.